database.py

```diff
@@ -5,30 +5,18 @@
 
 def create_table():
     with sqlite3.connect(db) as conn:
-        conn.execute('CREATE TABLE IF NOT EXISTS records(restaurant TEXT, recipe TEXT, flickr TEXT, bookmarked INTEGER)')
+        conn.execute('CREATE TABLE IF NOT EXISTS records(restaurant TEXT, recipe TEXT, flickr TEXT)')
     conn.close()
 
 def add_data(yelp_response, edamam_response, flickr_response):
     bookmarked = 0
     try:
         with sqlite3.connect(db) as conn:
-            conn.execute('INSERT INTO records (restaurant, recipe, flickr, bookmarked) VALUES (?, ?, ?, ?)', (yelp_response, edamam_response, flickr_response, bookmarked))
+            conn.execute('INSERT INTO records (restaurant, recipe, flickr, bookmarked) VALUES (?, ?, ?)', (yelp_response, edamam_response, flickr_response))
```

Comment:
@whagestad You have bookmarked as variable and but you removed it from the values

Reply:
Ok, fixed that.